Foundation-Mission-Structure-and-Strategy-Proposal.md

```diff
@@ -0,0 +1,76 @@
+# Node.js Foundation Mission, Structure and Strategy
+
+_Proposal to the Node.js Foundation TSC and Board, by Rod Vagg_
+
+**Goal #1:** Agree on a basic statement of mission for the Foundation, upon which all execution, by each group within the Foundation, can be tested and justified.
+
+**Goal #2:** Agree on a strategy generation and execution structure, and the relationship between the main bodies within that structure, that can best achieve the mission of the Foundation.
+
+## Foundation Mission Statement
+
+**The Node.js Foundation exists to increase the adoption of Node.js.**
```

Comment:
Along the same  line of thought I wonder if adoption is just one measure of success.  The mission could be:

"The Node.js Foundation exists to make Node.js the best runtime for development and deployment of applications with one measure of success being the level of adoption in open source and commercial communities and its market share relative to other runtimes"